pkg/controllers/managementuser/rbac/podsecuritypolicy/serviceaccount.go

```diff
@@ -118,12 +118,12 @@ func (m *serviceAccountManager) sync(key string, obj *v1.ServiceAccount) (runtim
 		return nil, nil
 	}
 
-	err := checkClusterVersion(m.clusterName, m.clusterLister)
+	err := CheckClusterVersion(m.clusterName, m.clusterLister)
 	if err != nil {
-		if errors.Is(err, errVersionIncompatible) {
+		if errors.Is(err, ErrClusterVersionIncompatible) {
 			return obj, nil
 		}
-		return obj, fmt.Errorf(clusterVersionCheckErrorString, err)
+		return obj, err
```

Comment:
If you are going to make changes to how this error is handled, I would recommend that you properly wrap the error, something like:

```suggestion
		return obj, fmt.Errorf("error when checking cluster version for serviceAccount controller: %w", err)
```

Reply:
Agreed, will adjust!